src/zkbob/ZkBobPool.sol

```diff
@@ -108,6 +136,14 @@ abstract contract ZkBobPool is IZkBobPool, EIP1967Admin, Ownable, Parameters, Ex
         _;
     }
 
+    /**
+     * @dev Throws if minimal tree update fee is not set.
+     */
+    modifier minTreeUpdateFeeIsSet() {
```

Comment:
Is this really necessary, if we will set the min fee during the upgrade anyway?

Reply:
Yes, indeed. It seems a bit excessive. Removed it in https://github.com/zkBob/zkbob-contracts/pull/97/commits/118287802305a9331f529cb19051b1e979220f9b.